summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -63,64 +72,72 @@ Fields:
 	
 ## Plots
 
 The results of this benchmark will be visualised in OpenEBench using the following plots:
 
-1. **bar plot** visualizing **Precision** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+1. **2D scatter plot** visualizing **Sensitivity and Precision** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
 
 - 10 nt
 - 50 nt
 - 100 nt
 
-**X axis** - benchmarked tool  
-**Y axis** - precision
+**X axis** - Sensitivity(d)  
+**Y axis** - Precision(d)  
+where _d - distance threshold_
 
 Input datasets:
 
 - RNA-Seq data compared with 3'end sequencing data
 - Simulated RNA-Seq data compared with dataset used for simulation
 
+Ranking: The best performing tool is the one with the highest Sensitivity combined with highest Precision (top right part of the plot) and the worst performing tool is the one with the lowest Sensitivity combined with lowest Precision (bottom left part of the plot).
+The plot should be divided into diagonal quartiles based on the distance from optimal performance.
+Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-right, bottom-right, top-left, bottom-left.
 
-Ranking: The best performing tool is the one with the highest precision value.
-
-2. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+2. **bar plot** visualizing **AUC** of Precision-Recall curve with single AUC value for each tool.
 
-- 10 nt
-- 50 nt
-- 100 nt
+**metric** - AUC
 
-**X axis** - FPR(d)  
-**Y axis** - TPR(d)  
-where _d - distance threshold_
-
-Input datasets:
+Input data:
 
 - RNA-Seq data compared with 3'end sequencing data
 - Simulated RNA-Seq data compared with dataset used for simulation
 
-Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
+Ranking: The best performing tool is the one with the highest AUC value.
 
-3. **bar plot** visualizing **AUC** of ROC plot with single AUC value for each tool.
+3. **bar plot** visualizing **proportion of PAS assigned to 3'-UTRs**.
 
-**X axis** - benchmarked tool  
-**Y axis** - AUC
+**metric** - proportion of PAS assigned to 3'-UTRs
 
 Input data:
 
 - RNA-Seq data compared with 3'end sequencing data
 
+Ranking: The best performing tool is the one with the highest proportion of PAS assigned to 3'-UTRs
```

Comment:
Thanks, that was something I wasn't sure about myself. I was also thinking of something along the lines of "in the 3'-UTR or a reasonable distance downstream from 3'UTR", but that's getting more vague. It will be easier to just add the plots without ranking and leave the interpretation to the reader.